.github/workflows/avm.res.desktop-virtualization.application-group.yml

```diff
@@ -0,0 +1,81 @@
+name: "avm.res.desktop-virtualization.application-group"
```

Comment:
Hi @moisesjgomez, also here, codeowners file should be updated. For guidelines on how to configure teams please refer to https://azure.github.io/Azure-Verified-Modules/specs/shared/#team-memberships

Reply:
I've created the teams and added the line in the codeowners file.